src/Illuminate/Validation/composer.json

```diff
@@ -16,6 +16,7 @@
     "require": {
         "php": "^8.0.2",
         "ext-json": "*",
+        "brick/math": "^0.10.2|^0.11",
```

Comment:
I don't think this should have been added here. Only as a suggest.

Reply:
This would have been a breaking change if it was only a suggest. Anyone currently using the `multiple_of` validation rule would have had issues on update.